apps/payments/nextjs/pages/embedded-checkout/v3.tsx

```diff
@@ -1,4 +1,4 @@
-import { CrossmintEmbeddedCheckout, CrossmintProvider } from "@crossmint/client-sdk-react-ui";
+import { CrossmintEmbeddedCheckout_Alpha, CrossmintProvider } from "@crossmint/client-sdk-react-ui";
```

Comment:
have to actually rename this in the react lib

Reply:
I was going crazy wondering what was going on. Turns out I was in the wrong directory when I `git add .`